index.html

```diff
@@ -102,7 +102,8 @@
 
 		<!-- banner start --> 
 		<div id="banner" class="banner">
-			<div class="banner-image"></div>
```

Comment:
To change background image you have to change the css file in styles.css. Change the image path in the banner-image class to change it